packages/custom-functions-metadata/test/cases/error-unsupported-input-parameter/expected.ts.errors.txt

```diff
@@ -0,0 +1 @@
+Invalid type: Excel.Worksheet (8,40)
```

Comment:
Rather than the message being `Invalid type`, it might be better to say that the type is not allowed for a custom function parameter.